### ISSUE TITLE
ERRO Aggregate attestation queue full

### ISSUE THREAD

**Issue by sealbox** [sigp/lighthouse#3584]
I am running lightouse version 3.1.0 and syncing from snapshot using following command:

```
/home/eth/node_beacon/lighthouse --network mainnet beacon_node --datadir=/home/eth/.ethereum --http --execution-endpoint=http://127.0.0.1:8551 --execution-jwt=/home/eth/node_beacon/jwt.hex --checkpoint-sync-url=XXX
```
It worked fine for few hours, got synced, but now it stopped syncing and is throwing following error:

```
lighthouse[32463]: Sep 16 12:13:21.443 ERRO Aggregate attestation queue full        queue_len: 4096, msg: the system has insufficient resources for load
```

What should I do?

**Comment by winksaville** [comment]
Just guessing, is your disk full or maybe insufficient RAM?

**Comment by sealbox** [comment]
No, my disk has plenty of space left and RAM is not utilized in 100%. I was also suspecting the system limit on open files handles, but I check soft , hard limits nad system.max-file, but everything is set much, much beyond 4096. I don't know what to do?

**Comment by michaelsproul** [comment]
This is indicative of a CPU or I/O bottleneck usually. It doesn't mean your node is inviable but it will probably lead to suboptimal validator performance if you are running validators.

Since the merge we've noticed several ways to improve performance which should help issues like this, and will have a series of releases coming soon to address them.

What hardware are you running on?

**Comment by AgeManning** [comment]
I suspect this is related to #3212 also. Once we have implemented a solution to that, I think that may also help.

**Comment by michaelsproul** [comment]
Closing as stale
